spec/suites/core/GeneralSpec.js

```diff
@@ -1,8 +1,10 @@
+import {expect} from 'chai';
+
 /* global L */
 import 'leaflet';
 
 describe('General', () => {
-	it('namespace extension', () => {
+	it.skip('namespace extension', () => {
```

Comment:
We could change the import map from `'leaflet': '/src/Leaflet.js',` to `'leaflet': '/src/LeafletWithGlobals.js',` but then the object is not [extendable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cant_define_property_object_not_extensible). A workaround could be following: `window.L = {...window.L}` but then the child objects are still not extendable and need to be overwritten with the workaround too `window.L.Util = {...window.L.Util}`. I don't think that it makes sense to test this, if we need a lot of workarounds